src/bin/cairo-native-run.rs

```diff
@@ -281,6 +308,35 @@ fn main() -> anyhow::Result<()> {
         }
     }
 
+    #[cfg(feature = "with-libfunc-counter")]
+    if let Some(libfunc_counter_output) = args.libfunc_counter_output {
+        use std::collections::HashMap;
+
+        let counters =
+            cairo_native::metadata::libfunc_counter::libfunc_counter_runtime::LIBFUNC_COUNTER
+                .lock()
+                .unwrap();
+        assert_eq!(counters.len(), 1);
+
+        let libfunc_counter = counters.values().next().unwrap();
```

Comment:
I do not fully get this part:

- Does `LIBFUNC_COUNTER` hold several counters?
- Why the `libfunc_counter` variable is set to the second value of `LIBFUNC_COUNTER`?

Reply:
Ok, makes sense